src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -415,7 +415,26 @@ public virtual bool RunInMemory
         [IndexUpdateType(IndexUpdateType.None)]
         [ConfigurationEntry("Indexing.Static.RequireAdminToDeployJavaScriptIndexes", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public bool RequireAdminToDeployJavaScriptIndexes { get; set; }
-
+        
+        [Description("Order by score automatically when boosting is inside index definition or query.")]
+        [DefaultValue(true)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.OrderByScoreAutomaticallyWhenBoostingIsInvolved", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public bool OrderByScoreAutomaticallyWhenBoostingIsInvolved { get; set; }
+        
+        [Description("EXPERT: Use compound file in merging")]
+        [DefaultValue(true)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.Lucene.UseCompoundFileInMerging", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        [ConfigurationEntry("Indexing.UseCompoundFileInMerging", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public bool LuceneUseCompoundFileInMerging { get; set; }
```

Comment:
Touching this setting makes that side-by-side index is created. Maybe that is because of `IndexUpdateType.None` instead of `IndexUpdateType.Refresh` ?

Reply:
Got fixed in https://github.com/ravendb/ravendb/pull/14755/commits/a9caccb500658e356f9955d8fe64bcf3fa20e87c